tests/subscriptions/test_scheduler_processing_strategy.py

```diff
@@ -410,14 +408,14 @@ def test_tick_buffer_with_commit_strategy_global() -> None:
     now = datetime.now()
 
     metrics_backend = TestingMetricsBackend()
-
     next_step = mock.Mock()
 
     strategy = TickBuffer(
         SchedulingWatermarkMode.GLOBAL,
         2,
         10,
         ProvideCommitStrategy(2, next_step, metrics_backend),
+        metrics_backend,
```

Comment:
from looking at this test I don't see that you've actually tested the new functionality you've introduced

Reply:
I haven't, this just passes the now-required metrics argument in. I don't think we should need to have unit tests over logging to datadog.

Reply:
We almost never have tests for stuff like this, and I'm not sure we want to start now. Unfortunately it's always the case that logging and metrics would have an outsize impact on any high throughput consumer.